ingestion/src/metadata/ingestion/source/dashboard/domodashboard/metadata.py

```diff
@@ -114,16 +114,16 @@ def yield_dashboard(
             )
             logger.debug(traceback.format_exc())
 
-    def get_page_details(self, page_id):
+    def get_page_details(self, page_id) -> dict:
```

Comment:
not sure if type hinting as `dict` works in Python 3.7. We might need to change this to `Dict[str, ...`]